### DISCUSSION TITLE
Scipy regression with anaconda progression?

### DISCUSSION THREAD

**Discussion by mforbes** [sagemathinc/cocalc#6388] [Q&A]
Why is the `scipy` version regressing with advancing `anaconda` packages?

* `anaconda2020` has python 3.7.8 and scipy 1.7.3
* `anaconda2021` has python 3.9.7 but scipy 1.7.1,
* `anaconda2022` (in the Ubuntu 22.04 environment) has python 3.9.15 but scipy 1.5.3! 

I was looking for something with scipy >= 1.8 (which brought in `scipy.sparse.lil_array`) or 1.10 which has been released.  (`anaconda2023`?)

**Comment by haraldschilly** [comment]
The pedantic answer has certainly something to do with the package resolver. I'll try to figure out if there is a blocking dependency.

**Reply by williamstein** [reply]
Oh, so you might be forced to remove some packages, since they are forcing back the scipy version? Makes sense.

**Comment by haraldschilly** [comment]
@mforbes finally had time to look into this. If you're curious, it's due to a pycaret dependency. Not sure why … made a note https://github.com/sagemathinc/cocalc/issues/6391 and sure, I'll see how much can be updated without that dragging everything back into the past.
